app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+html {
+    background-color: #343535b0;
+}
+
+.body-text {
+    background-color: rgb(209, 207, 207);
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+li {
+    list-style-type: none;
+    margin: 0 2vw;
+    font-size: 2vh;
+}
+
+nav {
+    width: 100vw;
+    background-color: #D6EFFF;
+}
+
+.header {
+    margin-left: 20px;
+    width: 100%;
+    text-decoration: none;
+    color: white;
+    padding: 2vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.link {
+    text-decoration: none;
+    color: black;
+    padding: 1vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+ul {
+    margin: 0;
```

Comment:
This is small but from looking at the css in User Portal there seems to be 2 spaces in their indentation whereas you have 4. Might be worth going with 2 spaces when it comes to css to keep it consistent with other css files with Wp Engine. This preference can vary from programming languages or even personal choice.

Reply:
This an ideal point to introduce the concept of linters 😄

Reply:
Wasn't sure if I should go there 😄 

@dylankwpe A tool called [Prettier](https://prettier.io/) is perfect for this situation. It's not so much a linter (which basically looks for errors in your code) but more-so a code formatter. It will automatically fix *most* of those annoying things that you may forget about or people often spend too much time talking about in code reviews. It will fix things like spacing, adding semi colons to the end of a line and some other things. What I like to do is run this automatically while I push to a repo meaning everyone is on the same page. No more fights about using 2 spaces or 4 spaces, Prettier just does it for you whether you like it or not 😄  We can go through this when you look at React if you want.

Reply:
Sounds handy af 😄 I used a linter before but couldn't get it to work very well so I'll wait to go through it with you when I go down the React route. I'm guessing it's a VSCode extension.

Reply:
Though you can get a VS Code extension, it's better to install it as part of your project so that:

1. Everyone contributing to the project has the same rules. If you and I had different rules in our vs code extensions it would cause inconsistencies. 
2. You have more control over running eslint when using it as a command line utility. For example running eslint during a pre-commmit hook.

All of this may be sound confusing. Don't worry, we will go through it when setting up the ReactJS project.